docs/naps/4-async-slicing.md

```diff
@@ -435,6 +483,12 @@ class ViewerModel:
         self._slicer.slice_layers_async(self.layers, self.dims)
 ```
 
+Initially we intend for each instance of `ViewerModel` to own a single
```

Comment:
@jni : clarified the one-to-one relationship between `ViewerModel` and `LayerSlice`.